homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -28,14 +28,8 @@ class YaleConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_import(self, config: dict):
         """Import a configuration from config.yaml."""
-        if not config[CONF_NAME]:
-            config[CONF_NAME] = DEFAULT_NAME
-        if not config[CONF_AREA_ID]:
-            config[CONF_AREA_ID] = DEFAULT_AREA_ID
 
-        self.context.update(
-            {"title_placeholders": {"name": f"YAML import {config[CONF_NAME]}"}}
-        )
+        self.context.update({"title_placeholders": {"name": f"YAML import {DOMAIN}"}})
```

Comment:
Nitpick: I see some components use constant `CONF_NAME` here, but most have hardcoded `"name"`.

Reply:
LOL. I had made some changes anyhow in the strings file so now I got an excuse to do another commit :)
Thanks for reviewing

Reply:
Reviewing this I learned about importing `configuration.yaml` into the UI, so quite useful for the integration I'm working on! (I've ported the `nobo_hub` integration to use async and added config and option flows.)